whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticAutocompleteServiceTestIntegration.java

```diff
@@ -464,6 +477,15 @@ public void filter_comment_multiline_value() {
         assertThat(getValues(query("AA1", "organisation", "org-name"), "org-name"), contains("Any"));
     }
 
+    @Test
+    @Disabled
```

Comment:
Add a `TODO` within the annotation explaining why it's disabled, e.g.
```
@Disabled("TODO: [MH] migrate from Lucene")
```

Reply:
@Disabled("TODO: [MH] migrated from Lucene, issue with ':' in ES")